chain/store/store.go

```diff
@@ -730,6 +734,55 @@ func (cs *ChainStore) readMsgMetaCids(mmc cid.Cid) ([]cid.Cid, []cid.Cid, error)
 	return blscids, secpkcids, nil
 }
 
+func (cs *ChainStore) GetPath(ctx context.Context, from types.TipSetKey, to types.TipSetKey) ([]*HeadChange, error) {
+	fts, err := cs.LoadTipSet(from)
```

Comment:
We should probably just reuse code from the `ReorgOps` method, it does a good hunk of this already

Reply:
Definitely, good that you mentioned it. It doesn't cover some cases I checked explicitly, but it's doing the important work (an error in those cases). Most of those things shouldn't happen in any reasonable scenario so it should be OK. I'll call that and then just do the remaining transformations.

Reply:
Do you think may worth to add the `build.ForkLengthThreshold` cap to `ReorgOps`?

Reply:
Probably worth adding that in, yeah. Maybe pass it as a parameter?